config/initializers/hcaptcha.rb

```diff
@@ -0,0 +1,6 @@
+if APP_CONFIG.dig(:hcaptcha, :enabled)
```

Comment:
since we're in modern ruby, you can also just `return` early from this file, i.e:

```ruby
return unless APP_CONFIG.dig(:hcaptcha, :enabled)

HCaptcha.configure do |...|
```